README.md

```
@@ -138,6 +138,10 @@ For more examples and detailed documentation, see the `examples/` directory.
 software quality information
 ----------------------------
 
+### Continuous Integration (Ubuntu, macOS, Windows)
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)